app/src/main/java/com/example/mymovieinfo/MainViewModel.kt

```diff
@@ -0,0 +1,19 @@
+package com.example.mymovieinfo
+
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.example.mymovieinfo.dto.Movie
+import com.example.mymovieinfo.service.MovieService
+import kotlinx.coroutines.launch
+
+class MainViewModel : ViewModel() {
+    var movies : MutableLiveData<List<Movie>> = MutableLiveData<List<Movie>>()
+    var movieService : MovieService = MovieService()
+    fun fetchCountries() {
+        viewModelScope.launch {
+            var innerMovies = movieService.fetchMovies()
+            movies.postValue(innerMovies)
+        }
+    }
+}
```

Comment:
This looks like an incomplete feature.
Are you using these anywhere?  I don't see them used.
If not in use, remove them.  Unused code is clutter, and bad documentation, which increases technical debt.
Only complete features, which meet the team's Definition of Done, should be merged to Main/Master.